src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -866,6 +870,32 @@ Prevents content overflow in navbar. */
     padding: 20px 40px;
 }
 
+#student-data-spreadsheet .panel-body {
+    padding: 5px;
+    margin: 10px;
+}
+
+#enrollstudents {
+    max-width: 100%;
+    opacity: 0;
+    height: 0;
+    margin: 0;
+    padding: 0;
+    outline: 0;
+}
+
+input[name='number_of_rows'] {
```

Comment:
Is there any particular reason why this element is selected with this expression instead of just giving it an id?